src/i18n/sv.js

```diff
@@ -0,0 +1,102 @@
+export default { 
+ error: { 
+   forgotPassword: { 
+     "too_many_requests": "Du har nått gränsen för maximalt antal försök att ändra ditt lösenord. Var vänlig vänta innan du försöker igen.", 
+     "lock.fallback": "Något gick fel när vi försökte ändra ditt lösenord." 
+   }, 
+   login: { 
+     "blocked_user": "Användaren är spärrad.", 
+     "invalid_user_password": "Fel på inloggningsuppgifter.", 
+      "lock.fallback": "Någit gick fel vid inloggningsförsöket.", 
+      "lock.invalid_code": "Fel kod.", 
+      "lock.invalid_email_password": "Fel email eller password.", 
+      "lock.invalid_username_password": "Fel användarnamn eller lösenord.", 
+      "lock.network": "Kunde inte kontakta servern. Var vänlig kontrollera din uppkoppling och försök igen.", 
```

Comment:
While not directly wrong, in Swedish it's more common to leave out the "please" part in an instruction, so I would suggest changing the last sentence to just "Kontrollera din uppkoppling och försök igen."